lib/actions/wallet.js

```diff
@@ -350,6 +374,18 @@ function joinWallet(walletId, options) {
     });
 
     dispatch(getWalletInfo(walletId, 'multisig'));
+
+    // find the cosigner with the token
+    // this should never hit the or case
+    const cosignerInfo = response.cosigners.find(c => c.token) || {};
+    dispatch(
+      setTemporarySecrets(walletId, {
```

Comment:
When joining or creating a multisig wallet, there are secrets that we want to temporarily display, but we don't want them to live around forever. the `joinKey` and `cosignerToken`